backend/package.json

```diff
@@ -28,6 +28,7 @@
   ],
   "scripts": {
     "start": "npm run build && node ./build/main.js",
+    "debug": "npm run build && node --inspect ./build/main.js",
```

Comment:
This is a new script I've been using that lets the chrome debugger attach to the BE. Here's a link to an [article for reference](https://medium.com/@paul_irish/debugging-node-js-nightlies-with-chrome-devtools-7c4a1b95ae27).